extensions/flutter-pub-dev-search/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pub-dev",
+  "title": "pub.dev",
+  "description": "Search pub.dev for packages and copy the latest version",
+  "icon": "command-icon.png",
+  "author": "prasadsunny1",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Find and install Dart/Flutter packages",
+      "subtitle": "Search for a Flutter package",
```

Comment:
```suggestion
      "subtitle": "pub.dev",
```

Reply:
Please also see our [naming guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)